integration-tests/features/steps/common.py

```diff
@@ -71,9 +73,10 @@ def migrate_vm_as_macrocontainer(context, source_vm, target_vm, migration_opt=No
     context.migration_target = target_vm
     context.migration_migration_opt = migration_opt
     migrate_app = context.cli_helper.migrate_as_macrocontainer
-    output = migrate_app(source_vm, target_vm, migration_opt)
+    output = migrate_app(source_vm, target_vm, migration_opt)  # nopep8
```

Comment:
In that case, it probably makes more sense to just drop the variable assignment from this line (thus avoiding the warning), and reword the following comment a bit:

    # TODO: Assert specifics regarding command output returned by the migrate_app() call